newsplease/crawler/commoncrawl_crawler.py

```diff
@@ -89,17 +90,34 @@ def __get_download_url(name):
     """
     return __cc_base_url + name
 
+def __iterate_by_month(start_date, end_date, month_step=1):
```

Comment:
This function and `newsplease/examples/commoncrawl.py:iterate_by` seem to be duplicates. Could you remove either of them? I would like to avoid redundant code in the project.